ts/screens/profile/RemoveAccountDetailsScreen.tsx

```diff
@@ -1,75 +1,157 @@
 import {
-  BlockButtonProps,
+  ButtonSolid,
   ContentWrapper,
-  FooterWithButtons
+  H6,
+  RadioGroup,
+  RadioItem,
+  TextInput,
+  VSpacer,
+  useIOToast
 } from "@pagopa/io-app-design-system";
-import * as pot from "@pagopa/ts-commons/lib/pot";
 import { StackActions } from "@react-navigation/native";
-import * as React from "react";
-import { Alert, SafeAreaView } from "react-native";
-import { connect } from "react-redux";
-import { Dispatch } from "redux";
-import { LabelledItem } from "../../components/LabelledItem";
-import { LoadingErrorComponent } from "../../components/LoadingErrorComponent";
-import {
-  RadioButtonList,
-  RadioItem
-} from "../../components/core/selection/RadioButtonList";
+import React, { memo, useCallback, useEffect, useMemo, useState } from "react";
+import { Alert, Keyboard, SafeAreaView, View } from "react-native";
+import { useSafeAreaInsets } from "react-native-safe-area-context";
 import { IOStyles } from "../../components/core/variables/IOStyles";
-import { RNavScreenWithLargeHeader } from "../../components/ui/RNavScreenWithLargeHeader";
 import { shufflePinPadOnPayment } from "../../config";
 import { isCgnEnrolledSelector } from "../../features/bonus/cgn/store/reducers/details";
 import I18n from "../../i18n";
 import NavigationService from "../../navigation/NavigationService";
 import { identificationRequest } from "../../store/actions/identification";
-import { navigateToWalletHome } from "../../store/actions/navigation";
 import {
   RemoveAccountMotivationEnum,
-  RemoveAccountMotivationPayload,
   removeAccountMotivation
 } from "../../store/actions/profile";
-import { GlobalState } from "../../store/reducers/types";
-import { userDataProcessingSelector } from "../../store/reducers/userDataProcessing";
+import {
+  isUserDataProcessingDeleteErrorSelector,
+  isUserDataProcessingDeleteLoadingSelector
+} from "../../store/reducers/userDataProcessing";
 import { withKeyboard } from "../../utils/keyboard";
+import { useIODispatch, useIOSelector } from "../../store/hooks";
+import { IOScrollViewWithLargeHeader } from "../../components/ui/IOScrollViewWithLargeHeader";
+import { useIONavigation } from "../../navigation/params/AppParamsList";
+import ROUTES from "../../navigation/routes";
+import { resetDeleteUserDataProcessing } from "../../store/actions/userDataProcessing";
 
-type Props = ReturnType<typeof mapStateToProps> &
-  ReturnType<typeof mapDispatchToProps>;
+type FooterButtonProps = {
+  isLoading: boolean;
+  onPress: () => void;
+};
 
-const getMotivationItems = (): ReadonlyArray<
-  RadioItem<RemoveAccountMotivationEnum>
-> => [
+const motivationItems: ReadonlyArray<RadioItem<RemoveAccountMotivationEnum>> = [
```

Comment:
We need to leave this as a function or move it inside the render function to avoid translations edge cases.  Am I right?